@navikt/core/css/darkside/guide-panel.darkside.css

```diff
@@ -1,43 +1,43 @@
 .navds-guide-panel {
-  --__ac-guide-panel-guide-size: 4rem;
+  --__axc-guide-panel-guide-size: 4rem;
 
   position: relative;
-  padding-top: calc(var(--__ac-guide-panel-guide-size) / 2);
-}
-
-.navds-guide {
-  background: var(--ac-guide-panel-illustration-bg, var(--a-surface-alt-3-subtle));
-  border: 2px solid var(--ac-guide-panel-border, var(--a-border-alt-3));
-  border-radius: var(--a-border-radius-full);
-  overflow: hidden;
-  position: absolute;
-  width: var(--__ac-guide-panel-guide-size);
-  height: var(--__ac-guide-panel-guide-size);
-  left: 50%;
-  transform: translate(-50%, -50%);
-}
-
-.navds-guide svg,
-.navds-guide img {
-  height: 100%;
-  width: 100%;
-}
+  padding-top: calc(var(--__axc-guide-panel-guide-size) / 2);
+
+  .navds-guide {
```

Comment:
Do you not think we should nest in the cases where we're not sure? (so most things get nested?)

Reply:
In this instance both guide and __content are 100% standalone, as the `.navds-guide-panel` selector makes no changes to them. So for this case i would compare it to just writing `.navds-guide-panel .navds-guide {}`, and increasing the specificity without a reason. Think it would make more sense if it was structured like this where the layout changes them
```
.navds-guide-panel[data-layout="poster"]{
  .navds-guide {}
  .navds-guide-panel__content {}
}

.navds-guide-panel[data-layout="not poster?"]{
  .navds-guide {}
  .navds-guide-panel__content {}
}
```

Reply:
yeah, I was thinking of refactoring it to use data props too, it became a bit confusing to think of the states & inverse states.

as for grouping / nesting: I don't want to have to lock myself in by being too specific (and then having to unnest / renest the CSS just because i ordered two html tags differently in the `.tsx`, currently this would happen if I moved components outside the top level one, but the top-most html element for a component could be sanctioned / special if we want to right? (there will always be at least one of these per component, and it can get the same name as the component itself). But variants at the top level (since a variant can affect a whole tree), could get their own top-level tree 🤔 (along with the media query getting its own).

To challenge the notion:

> In this instance both guide and __content are 100% standalone

In my head, `__content` is a logical "part" of the whole which is `.navds-guide-panel`, so for some reason I want that in the CSS too. And I also want a generic pattern that we can apply to all instances (to hopefully reduce cognitive load).

I might be overeager here in nesting. It's a tool, and if it's silly to nest too much, then we won't bother. I guess I just want some convention that leaves the fewest trees at the top level ( it can have a bunch of stuff inside each though) since I _think_ that could be neat and tidy (you can narrow down so much CSS by determining which tree you want to look in.)

Reply:
The code now looks like what you suggested (top level tree on layouts & media queries, I think we're actually in agreement?! 😳 ), so I'll resolve this.